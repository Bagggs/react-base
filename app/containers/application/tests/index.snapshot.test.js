import React from 'react';
import renderer from 'react-test-renderer';
import withStore from 'helpers/store';
import Application from 'containers/application';

describe('Application', () => {
  it('renders correctly', () => {
    const component = (
      <Application>
        <div />
      </Application>
    );
    const state = {
      modal: {},
      session: {
        loggedIn: false,
        currentUser: {}
      }
    };
    const applicationComponent = renderer.create(withStore(component, state));

    expect(applicationComponent.toJSON()).toMatchSnapshot();
  });
});
