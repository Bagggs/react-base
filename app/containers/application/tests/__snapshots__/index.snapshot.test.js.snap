// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Application renders correctly 1`] = `
<div
  className="layout"
>
  <main
    className="wrapper"
  >
    <nav
      className="panel navbar navbar-default"
    >
      <div
        className="container"
      >
        <div
          className="navbar-header"
        >
          <span
            className="navbar-brand"
          >
            React-base
          </span>
        </div>
        <ul
          className="nav navbar-nav"
        >
          <li>
            <a
              onClick={[Function]}
              style={Object {}}
            >
              Home
            </a>
          </li>
          <li>
            <a
              onClick={[Function]}
              style={Object {}}
            >
              About
            </a>
          </li>
        </ul>
        <ul
          className="nav navbar-nav navbar-right"
          role={null}
        >
          <li
            className=""
            role="presentation"
            style={undefined}
          >
            <a
              href="#"
              onClick={[Function]}
              role="button"
            >
              Sign up
            </a>
          </li>
          <li
            className=""
            role="presentation"
            style={undefined}
          >
            <a
              href="#"
              onClick={[Function]}
              role="button"
            >
              Sign in
            </a>
          </li>
        </ul>
      </div>
    </nav>
    <div
      loggedIn={false}
    />
  </main>
  <footer
    className="footer"
  >
    <p
      className="footer-text"
    >
      React-base
    </p>
  </footer>
  <div
    className="modals"
  />
</div>
`;
